src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -3358,7 +3358,10 @@ void ViewProviderSketch::camSensCB(void* data, SoSensor*)
     auto vp = proxyVPrdr->vp;
     auto cam = proxyVPrdr->renderMgr->getCamera();
 
-    vp->onCameraChanged(cam);
+    if (cam == nullptr)
+        Base::Console().DeveloperWarning("ViewProviderSketch", "Camera is nullptr!\n");
```

Comment:
> Is there an advantage in an explicit comparison rather than `if (!cam)`?


Yes, I feel more comfortable if I write in this fashion.


> Also, perhaps the warning should be slightly more helpful (or rather less intimidating for a non-developer); something for post 1.0.


Of course, could you suggest your message? I don't mind if somebody improves the text.

Reply:
Probably `"updateColorProperty: No Camera Found!"`?